blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx

```diff
@@ -26,6 +26,9 @@ function convertJenkinsNodeDetails(jenkinsNode, isCompleted) {
     } else if (jenkinsNode.result === 'FAILURE') {
         state = 'failure';
         completePercent = 100;
+    } else if (jenkinsNode.result === 'UNSTABLE') {
```

Comment:
I think that's probably less clear as to what's going on, and also it means that should the constants change we would then also have to change the program flow, which rubs me the wrong way, because I am a sensitive little flower :)